addon/validators/base.js

```diff
@@ -85,25 +85,24 @@ export default Ember.Object.extend({
    * @param  {Object} context     : Context for string replacement
    * @return {String}             : The generated message
    */
-  createErrorMessage(type, options, value, context) {
+  createErrorMessage(type, options, value, context = {}) {
     options = options || {};
```

Comment:
its such a small thing, I am indifferent. I just wanted to point out the difference.

You should choose based on impact to users in the wild. If it wont cause much (or any grief) go for it, if it causes some. It may not be worth it.